docs/source/api.rst

```diff
@@ -1,2 +1,70 @@
-Using the API
-=============
+Using the GameBench API Client
+==============================
+
+Installation
+------------
+
+The GameBench API Client can be install using pip:
+
+.. code-block:: python
+
+    pip install gamebench-api
+
+It can also be found on the `GitHub`_ page.
+
+
+Setup
+-----
+
+Add the username and password for the GameBench account you are using to the 'global_settings.py'
+module.
+
+.. code-block:: python
+   :linenos:
+
+    GAMEBENCH_CONFIG = {
+        'url': 'https://api.production.gamebench.net',
+        'api_version': '/v1',
+        'username': 'john.smith@example.com',
+        'password': 'password',
+        'company_id': '',
+    }
+
+See :doc:`global_settings` for for information on this module.
+
+
+Importing
+---------
+
+In the module that will be calling the API Client, import the ModelCreator class.
+
+.. code-block:: python
+
+    from gamebench_api_client.models.creator.model_creator import ModelCreator
+
+Create an instance of the ModelCreator and pass in the following information:
+
+ - The model that you want, which should be in a CamelCase style.
+ - A dictionary that specifies the information you want.
+
+Here is one example for requesting time series data:
+
+.. code-block:: python
+   :linenos:
+
+   time_series_request = {
```

Comment:
Do all of these items need to be in the dictionary if only the metric and session_id are used? Also, let's get a story to eliminate the duplication of model specification in the arguments. The metric value and the model name both communicate the same thing.

Reply:
The 'detail' key is the only one that doesn't need to be there for all of them.  It is only used in session_detail requests.  The others are important for how we are creating the attributes for Requests.  We have logic specifically looking for those keys.

For the model/metric duplication, those currently both have to be in there.  For most of the requests those will be the same, but there are ones where they will be different.  One of these  is when requesting the SessionSummary, which would be passed in as the 'model' argument.  However, in this case the 'metric' key would be left blank because that key is what will be added onto the URL by the builder.  

To make this easier on the user we could do something where the model requested adds the value to 'metric' so they don't have to pass that into the dictionary.

Reply:
Cool! We talked through this and think a few stories to reduce the argument burden on the client would be good. Essentially, the models can be responsible for adding the metric path or any other element that it cares about. The request builder can inject empty elements if none are provided. Finally, we might remove the session detail models in favor of pointers on session summary objects to the dataframe elements of the details.